core/commands/store/upload/upload.go

```diff
@@ -198,7 +198,7 @@ Use status command to check for completion:
 					host, err := hp.NextValidHost(price)
 					if err != nil {
 						f.Error(err)
-						return err
+						return nil
```

Comment:
Maybe it shouldn't return err, the only err `NextValidHost` returns is to indicate end of hosts. Maybe return `(string, bool)`

Reply:
will do this in BTFS-1603